### DISCUSSION TITLE
Is there a jacoco plugin?

### DISCUSSION THREAD

**Discussion by ice1000** [com-lihaoyi/mill#1443] [Q&A] [answered]
Jacoco is the Java code coverage tool.

**Comment by lefou** [comment]
Non yet, but I wrote one for an non-public project. If there is demand, I can make it public and standalone, though.

**Reply by ice1000** [reply]
Cool!!

**Comment by lefou** [comment, marked as answer]
I just published https://github.com/lefou/mill-jacoco.
